### PR TITLE
Rebranding to Yivi, including texts and bump package versions

### PR DESCRIPTION
**Pull request by bobhageman**

### CODE REVIEW

**Review by ivard — COMMENTED** [review]
Please wait merging until the name change date comes close.

**Comment by ivard** [comment]
Please wait merging until the name change date comes close.

**Comment by ivard** [comment]
Please wait merging until the name change date comes close.

File: src/main/resources/config.sample.json [hunk 1]
```diff
@@ -12,12 +12,12 @@
...
   "sms_templates": {
-    "nl": "IRMA verificatecode: %s\nOf direct via een link:\nhttps://privacybydesign.foundation/uitgifte/telefoonnummer#!verify:%s",
-    "en": "IRMA verification code: %s\nOr directly via a URL:\nhttps://privacybydesign.foundation/issuance/phonenumber#!verify:%s"
+    "nl": "Yivi verificatecode: %s\nOf direct via een link:\nhttps://privacybydesign.foundation/uitgifte/telefoonnummer#!verify:%s",
+    "en": "Yivi verification code: %s\nOr directly via a URL:\nhttps://privacybydesign.foundation/issuance/phonenumber#!verify:%s"
   },
   "private_key_path": "sk.der",
   "server_name": "irma_sms_issuer",
-  "human_readable_name": "Privacy by Design Foundation",
+  "human_readable_name": "Yivi",
```
Comment (ivard) [inline src/main/resources/config.sample.json:20]: Please wait merging until the name change date comes close.

**Review by sjoerdlmkns — APPROVED** [review]

**Pull request merged**